src/Component/DOMElementList.php

```diff
@@ -25,7 +25,7 @@ final class DOMElementList implements IteratorAggregate, Countable {
 	 *
 	 * @var DOMElement[]
 	 */
-	private $elements = [];
+	public $elements = [];
```

Comment:
My mistake, https://github.com/ampproject/amp-wp/pull/3659/commits/38df209f9b66a3e0ce9eefff468d527c5cd8ce1d changes it back to `private`.

I originally changed it because of failing unit tests, but they pass now.